altos-rust/port/cortex-m0/src/peripheral/usart/baudr.rs

```diff
@@ -12,27 +35,61 @@ pub enum BaudRate {
     Hz115200,
 }
 
-#[derive(Copy, Clone)]
+/// Stores base address of the BRR, which is the address
+/// of the Usart being used to access this register.
+#[derive(Copy, Clone, Debug)]
 pub struct BRR {
     base_addr: *const u32,
 }
 
+/// Implements the Register trait for Usartx_BRR.
+/// Stores base address for the baud rate register, which is the address
+/// of the Usart being used to access this register. Uses the base address
+/// combined with the register offset to calculate the register address.
 impl Register for BRR {
+    /* Sets the base address as the Usart address.
+     * Returns itself to the calling routine.
+     */
```

Comment:
I think we can just leave out comments here in general. In the generated docs it will just link to any implemented traits automatically